File: FinalProject_Test/app/src/main/java/com/example/finalproject_test/DATA/Models/Ranking.java
package com.example.finalproject_test.DATA.Models;

public class Ranking {
    private String username;
    private int point;

    private User usernameNavigation;

    public Ranking(String username, int point) {
        this.username = username;
        this.point = point;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public int getPoint() {
        return point;
    }

    public void setPoint(int point) {
        this.point = point;
    }

    public User getUsernameNavigation() {
        return usernameNavigation;
    }

    public void setUsernameNavigation(User usernameNavigation) {
        this.usernameNavigation = usernameNavigation;
    }
}

File: API_QuizAppDB/Models/Ranking.cs
using System;
using System.Collections.Generic;

namespace API_QuizAppDB.Models
{
    public partial class Ranking
    {
        public string Username { get; set; } = null!;
        public int? Point { get; set; }

        public virtual User UsernameNavigation { get; set; } = null!;
    }
}
